member.php

```diff
@@ -1267,9 +1267,14 @@
 
 			$validator_javascript .= "
 	$.validator.addMethod('passwordSecurity', function(value, element, param) {
-		return (value != $('#email').val() && value != $('#username').val()
-			&& value.indexOf($('#email').val()) === false && value.indexOf($('#username').val()) === false
-			&& $('#email').val().indexOf(value) === false && $('#username').val().indexOf(value) === false);
+		return !(
+				$('#email').val() != '' && value == $('#email').val()
```

Comment:
I wonder if it makes sense to break each of these branches out into individual variables to make it easier to read and follow (and prevent similar errors in the future)...

Reply:
Quite possibly, although it would end up looking pretty much the same here so comments might be a more practical idea?

Reply:
Sure, comments work too.